extensions/http/src/main/java/jolie/net/http/HttpMessage.java

```diff
@@ -90,10 +89,10 @@ public boolean secure() {
 	private Version version;
 	private final Type type;
 	private byte[] content = null;
-	final private Map< String, String > propMap = new HashMap<>();
-	final private List< Cookie > setCookies = new ArrayList<>();
```

Comment:
Why spaces have been previously introduced and now they are being deleted? Is something wrong with your formatter configuration?